spec/yaml/swiftnav/sbp/settings.yaml

```diff
@@ -17,39 +17,55 @@ description: |
   the host request and the device response. See the accompanying
   document for descriptions of settings configurations and examples:
   https://github.com/swift-nav/piksi\_firmware/blob/master/docs/settings.pdf
-stable: False
+stable: True
 public: True
 include:
   - types.yaml
 definitions:
 
- - MSG_SETTINGS:
-    id: 0x00A0
-    public: True
-    short_desc: |
-      R/W device configuration settings (host <=> device)
+ - MSG_SETTINGS_SAVE:
+    id: 0x00A1
+    short_desc: Save settings to flash (host => device)
     desc: |
-      The setting message reads and writes the device's configuration.
+      The save settings message persists the device's current settings
+      configuration to its onboard flash memory file system.
+
+ - MSG_SETTINGS_WRITE:
```

Comment:
Oh, hm, that' a good question. The preceding behavior was echoing back the write (it was multipurpose and also being used for reads). But then the usage of it in the console was just super weird and bad - I just removed the return message and things work fine. It's just super weird around this call now. What do you think?